README.md

```diff
@@ -45,7 +45,32 @@ while builds_response.next_page:
     builds_response = buildkite.builds().list_all(page=builds_response.next_page, with_pagination=True)
 ```
 
+## Artifacts
 
-# License
+Artifacts can be downloaded as binary data. The following example loads the artifact into memory as
+[Python bytes](https://docs.python.org/3/library/stdtypes.html#binary-sequence-types-bytes-bytearray-memoryview)
+and then writes them to disc:
+
+```python
+artifacts = buildkite.artifacts()
+artifact = artifacts.download_artifact("org_slug", "pipe_slug", "build_no", 123, "artifact")
+with open('artifact.bin', 'b') as f:
+  f.write(artifact)
```

Comment:
Is there specific reason the examples seem to be writing to the downloaded artifact? Would it make more sense to read data from it?

Reply:
This example writes the binary data that are stored in variable `artifact` into file `"artifact.bin"` via file handler `f`. Writing to the file is an example of how to consume the binary data of the artifact.

Reply:
Makes plenty of sense.